lib/crypto.js

```diff
@@ -594,7 +595,7 @@ ECDH.prototype.getPublicKey = function getPublicKey(encoding, format) {
     else if (format === 'uncompressed')
       f = constants.POINT_CONVERSION_UNCOMPRESSED;
     else
-      throw new TypeError('Bad format: ' + format);
+      throw new error.TypeError('ERR_INVALID_FORMAT', format);
```

Comment:
I am not too happy with a new error code for this single line. How about `ERR_INVALID_OPT_VALUE`?